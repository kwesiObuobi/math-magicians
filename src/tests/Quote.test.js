import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Quote from '../components/quote';

describe('Navbar Component', () => {
  it('renders the Navbar component', async () => {
    render(<Quote />);
    expect(screen.getByText('Loading...')).toBeInTheDocument();
    expect(screen.queryByRole('link')).toBeNull();
  });

  it('should correctly render the Navbar component', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
