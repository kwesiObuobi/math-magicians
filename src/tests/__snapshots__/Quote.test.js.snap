// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Component should correctly render the Navbar component 1`] = `
<div
  className="quote"
>
  Loading...
</div>
`;
